pymatgen/io/vasp/tests/test_inputs.py

```diff
@@ -881,6 +881,37 @@ def test_keywords(self):
         }
         self.assertEqual(self.psingle.keywords, data)
 
+    def test_psctr(self):
+        filename = PymatgenTest.TEST_FILES_DIR / "POT_GGA_PAW_PBE_54" / "POTCAR.Fe.gz"
+
+        with pytest.warns(None):
```

Comment:
I think this has been deprecated:

> E   pytest.PytestRemovedIn8Warning: Passing None to catch any warning has been deprecated, pass no arguments instead:
E    Replace pytest.warns(None) by simply pytest.warns().

Reply:
Two pytest.warns(None) in test_inputs.py are replaced by pytest.warns() in the new commit.